api/init/graphqlapi/data_source.py

```diff
@@ -0,0 +1,34 @@
+import docker
+import logging
+import graphqlapi.utils as utils
+
+
+# Load logging configuration
+log = logging.getLogger(__name__)
```

Comment:
@ThomasGassmann I guess we can remove the logger here. It's not used in the module.

Reply:
Updated in 604db3aaad70597e1fa7a7e9978086ec9995c817.

Reply:
@ThomasGassmann could you please remove the `import logging` too?

Reply:
fixed @alexisrolland